layouts/default.vue

```diff
@@ -25,6 +14,29 @@ onMounted(() => {
 
 <template>
   <div class="page" :class="[`${route.name}`]" :data-style-mode="darkMode ? 'dark' : 'default'">
+    <Html>
+      <Head>
+        <Title>Radiolab: Podcasts | WNYC Studios | Podcasts</Title>
+        <Meta name="description" content="Investigating a strange world." />
+        <Meta name="keywords" content="podcasts, npr, new york, WNYC Studios, arts, culture, classical, music, news, public, radio" />
+        <Meta name="og:site_name" content="Radiolab Podcasts | WNYC Studios" />
+        <Meta name="og:type" content="website" />
+        <Meta name="og:url" :content="`https://www.radiolab.org${route.fullPath}`" />
+        <Meta name="og:title" content="Radiolab: Podcasts | WNYC Studios | Podcasts" />
+        <Meta name="og:description" content="Investigating a strange world." />
+        <Meta name="og:image" content="https://media.wnyc.org/i/1200/600/l/80/2020/09/Radiolab_OG_Image_Options_3.png" />
+        <Meta name="og:image:width" content="1200" />
+        <Meta name="og:image:height" content="600" />
```

Comment:
for some reason, the going trend is the OG image be 1.9:1 ratio. So height would be 630. Be sure to update the image url as well.

Reply:
seems like any system that is consuming these images manipulates them anyway... so might as well provide them with the best we got to work with. So just ignore all the image updates.